sensors/slack_sensor.py

```diff
@@ -158,6 +162,13 @@ def _handle_message(self, data):
             # Note: We resolve user and channel information to provide more context
             user_info = self._get_user_info(user_id=data['user'])
 
+        # Check if the message was written by a bot without as_user attribute
+        if self._allow_bot_messages and 'user' not in data and 'username' in data:
+            user_info = {'name': data['username'], 'profile': {}}
+        else:
+            # Note: We resolve user and channel information to provide more context
+            user_info = self._get_user_info(user_id=data['user'])
+            
```

Comment:
This line contains whitespace - that's why the build failed: https://circleci.com/gh/StackStorm-Exchange/stackstorm-slack/110#action-104

Please remove it.

Reply:
wired, its like that master too. its not a line I changed
https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/sensors/slack_sensor.py#L171

Reply:
Done